python/paddle/v2/fluid/distribute_transpiler.py

```diff
@@ -457,27 +566,33 @@ def get_pserver_program(self, endpoint):
             pserver_program.global_block().create_var(
                 name=v.name, persistable=True, dtype=v.dtype, shape=v.shape)
             for trainer_id in xrange(self.trainers):
-                print("create variable for program: %s.trainer_%d" %
-                      (v.name, trainer_id))
                 pserver_program.global_block().create_var(
                     name="%s.trainer_%d" % (v.name, trainer_id),
                     persistable=True,
                     dtype=v.dtype,
                     shape=v.shape)
         # step6
         optimize_sub_program = Program()
+        ufind = self._create_ufind(self.optimize_ops)
```

Comment:
Add comments or document about why we need `ufind`.

Reply:
Done.